packages/sui-studio/src/components/demo/index.js

```diff
@@ -66,7 +64,7 @@ export default class Demo extends Component {
       playground,
       style,
       themes,
-      ctxt: context,
+      ctxt: context
```

Comment:
disagree with this, I think it should end with comma as later on it's easier to update